### ISSUE TITLE
Migrate Knowledge Base articles to the unified public documentation

### ISSUE THREAD

**Issue by yanesca** [Mbed-TLS/mbedtls#4073]
## Background

As a consequence of the transition to TF.org we would like to move to the new infrastructure and as part of this we are going to migrate the information available on the website. The goal of this task is to migrate Knowledge Base articles. Migrating security advisories is tracked in #3211 and is out of scope for this task.

This migration has already been done to https://github.com/ARMmbed/mbed-tls-docs and there were further updates. Therefore it is best to take the KB articles from this repository, ideally preserving git history as well.

We also need to help users find the articles at their new place and should add a message to the header of the knowledge base pages on the website, preferably with a link to the new location.

**Prerequisite:** #3643

## Task

The task is done when 

- [x] all existing knowledge base articles from the website are copied to the documentation site (https://github.com/ARMmbed/mbedtls-docs)
- [x] the knowledge base article template on the website has a header with an announcement and redirecting to the ReadTheDoc page

**Comment by yanesca** [comment]
An efficient way to do it would be to ask the devops team to retrieve the markdown sources from the database and submit them in a PR.

**Comment by yanesca** [comment]
Editing and updating the articles is out of scope. Some minimal editing is best done as a quick followup and is tracked here:
https://github.com/ARMmbed/mbedtls/issues/4074

**Comment by gilles-peskine-arm** [comment]
> the knowledge base article template on the website has a header with an announcement and redirecting to the ReadTheDoc page

This is referring to the old website that isn't live anymore and the URL redirects to readthedocs. So I guess this issue is fully done?

**Comment by yanesca** [comment]
Indeed, this is fully done and we can close it down.
